geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/index/FilterHelperTest.scala

```diff
@@ -353,4 +356,15 @@ class FilterHelperTest extends Specification with Mockito with Logging {
       result.asInstanceOf[KnownCost].cost mustEqual 5
     }
   }
+
+  "tryMergeGeoms" should {
+    "intersect BBOX geoms" >> {
```

Comment:
Additionally, there should be a test cases where 1) the intersection is not just one of the boxes, and 2) involving the IDL and a view port spanning the IDL.